django/cantusdb_project/main_app/views/feast.py

```diff
@@ -96,7 +97,8 @@ def get_ordering(self):
         # default to ordering by name if given anything else
         if ordering not in ["name", "feast_code"]:
             ordering = "name"
-        return ordering
+        # case insensitive ordering by name
+        return (Lower(ordering) if ordering == "name" else ordering, )
```

Comment:
curious about the `,` at the end of `ordering,` - does `get_ordering` need to return a tuple? If so, maybe a type annotation on `get_ordering` would clarify things, as well as wrapping `ordering,` in parentheses: `return (... else (ordering,))`.